server/template.js

```diff
@@ -17,7 +17,7 @@ try {
 
 export default ({ url, bodyClass, title, description }) => marinate`
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
we do want to support localization at some point. would we change this programmatically or use a variable we read from somewhere?

Reply:
Thanks for suggesting it @tsingh777, We definitely need to have such a thing in place for longer support, but before adding a variable we need to decide on a URL pattern for language support. Not sure if we have something already in place?
**Example**
`sikhitothemax.org/en/`
`sikhitothemax.org/en-US/`

However, I added a TODO comment for the future.

Reply:
Not sure if we pattern yet for localization. Some options:
- Add a selector somewhere on the UI and store in a cookie/localstorage and read from that.  <- my preference 
- auto detect from browser locale 

Might need to refactor a little bit so we can use something like https://www.npmjs.com/package/react-intl or https://github.com/lingui/js-lingui

Reply:
thanks for your suggestion @tsingh777 !
I added a cookie to save the language, and in case it is not available fallback will go to "en" which is also defined as constant.